### PR TITLE
User interaction

### PR DESCRIPTION
**Pull request by stefanleminh**
This pull requests adds the functionality of being able to move the users to the designated voice channels once a match starts or ends.

### CODE REVIEW

File: commands/randomize.js [hunk 1]
```diff
@@ -2,15 +2,21 @@ const Discord = require("discord.js");
 const functions = require('../modules/functions')
 
 exports.run = (client, message, args) => {
+  if(client.voiceChannels.length === 0) {
+    message.channel.send("You have not started a session yet! Please run the =newsession command.")
+    return
+  }
```
Comment (AndreasMReumschuessel) [inline commands/randomize.js:8]: This check is popping up in almost every command. We should introduce a helper class for validation checks like these, so we avoid duplicate code and improve readability.
I'd think of something like
```
if(!isActiveSession(client)) {
   return;
}
```

File: commands/endsession.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 exports.run = (client, message, args) => {
+    if(client.voiceChannels.length === 0) {
+        message.channel.send("You have not started a session yet! Please run the =newsession command.")
+        return
+      }
     client.sessionRunning = false;
```
Comment (AndreasMReumschuessel) [inline commands/endsession.js:6]: Is this still in use? The session check is based on the `client.voiceChannels.length === 0` check and this variable is only set to false here and in index.js.

**Review by AndreasMReumschuessel — CHANGES_REQUESTED** [review]
Good work 👍! Just some minor things to imrove I'd say.
It is a bit confusing to handle the team lists and spectator lists instead of having a pool of players with attributes in which team they are and if they are spectator or not. But that's another story :D

File: commands/endsession.js (hunk 2 above)
Comment (stefanleminh) [reply commands/endsession.js:6]: You're right, the boolean one is not needed anymore, will be removed in the next commit.

**Review by AndreasMReumschuessel — APPROVED** [review]
Mixed feelings about this decision, but let's face it in another issue because I think we should have persistent configurations sooner or later :)

File: commands/newsession.js [hunk 3]
```diff
@@ -3,26 +3,11 @@ exports.run = (client, message, args) => {
...
-    if(client.voiceChannels.some(channel => channel.name === channelName)) {
-      message.channel.send(
-        "Channel " + channelName + " is already on the list!"
-      );
-    }
-    client.voiceChannels.push(channel);
-  });
+  client.voiceChannels.push(message.guild.channels.cache.get(client.config.lobby))
+  client.voiceChannels.push(message.guild.channels.cache.get(client.config.firstTeamVc))
+  client.voiceChannels.push(message.guild.channels.cache.get(client.config.secondTeamVc))
```
Comment (AndreasMReumschuessel) [inline commands/newsession.js:9]: Could you also add it to the blank config, please?

**Review by AndreasMReumschuessel — CHANGES_REQUESTED** [review]

File: commands/newsession.js (hunk 3 above)
Comment (stefanleminh) [reply commands/newsession.js:9]: Done!

File: commands/randomize.js (hunk 1 above)
Comment (stefanleminh) [reply commands/randomize.js:8]: Resolved in revision a20d2f66596e12161ac86cf656e57160db4cfe7e 
Is this how you imagined it?

Comment (AndreasMReumschuessel) [reply commands/randomize.js:8]: Yes!

File: commands/endmatch.js [hunk 4]
```diff
@@ -6,15 +6,18 @@ exports.run = (client, message, args) => {
...
-      const member = message.guild.members.cache.get(player.id);
-      member.voice.setChannel(client.voiceChannels[0])
-    });
+    if(client.voiceChannels[1].members.size > 0) {
+      client.voiceChannels[1].members.array().forEach((player) => {
+        player.voice.setChannel(client.voiceChannels[0])
+      })
+    }
+    
+    console.log(client.voiceChannels[2].members.values())
```
Comment (AndreasMReumschuessel) [inline commands/endmatch.js:15]: Is this log statement still needed? If not, remove it.

**Review by AndreasMReumschuessel — CHANGES_REQUESTED** [review]

File: commands/endmatch.js (hunk 4 above)
Comment (stefanleminh) [reply commands/endmatch.js]: Resolved in Revision 8a85f36f800986214cbff6dea4076e472d0123d4

**Pull request merged**